Makefile

```diff
@@ -19,10 +19,26 @@ run:
 		${CMD}
 .PHONY: run
 
+# all services
 serve:
-	docker-compose up
+	docker compose --profile all -f docker-compose.yml -f docker-compose.full.yml up
```

Comment:
Does the order matter? do we need slim/full first, then the main one?

Reply:
my understanding is they're combined into one file when ran so order does matter due to overrides but I think this is the correct way round and currently doesn't matter?
https://docs.docker.com/engine/reference/commandline/compose/#specifying-multiple-compose-files

Reply:
I was thinking more with the order things start in, but if this works that's all good.